bin/doctest

```diff
@@ -4,7 +4,7 @@
 Program to execute doctests using the py.test like interface.
 
 The advantage over py.test is that it only depends on sympy and should just
-work in any circumstances. See "sympy.dotest?" for documentation.
+work in any circumstances. See "sympy.doctest?" for documentation.
```

Comment:
Good catch.  I'd say this should be `help(sympy.doctest)`, the question mark syntax is specific to Jupyter notebooks, and I suppose most people running tests work in the console. `help` works everywhere.